### PR TITLE
adding key handover runbook

### PR DESCRIPTION
**Pull request by sbvegan**
**Description**

The runbook for creating key handover tasks

### CODE REVIEW

**Comment by sbvegan** [comment]
@zchn yep thats our expectation, thanks! I just wanted to make it easily accessible. We'll have the ready for review sometime next week.

File: runbooks/key-handover/SignFromJson.s.sol [hunk 1]
```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {SignFromJson as OriginalSignFromJson} from "script/SignFromJson.s.sol";
+import {ProxyAdmin} from "@eth-optimism-bedrock/src/universal/ProxyAdmin.sol";
+import {console2 as console} from "forge-std/console2.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {Vm, VmSafe} from "forge-std/Vm.sol";
+
+contract SignFromJson is OriginalSignFromJson {
```
Comment (blmalone) [inline runbooks/key-handover/SignFromJson.s.sol:10]: I'm not sure we want to be duplicating code in the runbooks directory. 

This file is already included as part of https://github.com/ethereum-optimism/superchain-ops/tree/main/tasks/eth/mmz-002-key-handover

File: runbooks/key-handover/VALIDATION.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Validation
+
+This document can be used to validate the state diff resulting from the execution of Key Handover process.
```
Comment (blmalone) [inline runbooks/key-handover/VALIDATION.md:3]: Not sure we want to be duplicating this `validation.md` file. 

We have the mmz one here which can be reused: https://github.com/ethereum-optimism/superchain-ops/blob/main/tasks/eth/mmz-002-key-handover/VALIDATION.md

File: runbooks/key-handover/.env [hunk 3]
```diff
@@ -0,0 +1,3 @@
+ETH_RPC_URL="https://ethereum-rpc.publicnode.com"
```
Comment (blmalone) [inline runbooks/key-handover/.env:1]: Similar comment on not duplicating, I think it's better to reuse code that's been through the PR process before.

File: runbooks/key-handover/README.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# {Chain Name i.e. Base Mainnet} Key Handover Upgrade
```
Comment (blmalone) [inline runbooks/key-handover/README.md:1]: Same comment on duplication.

File: runbooks/key-handover/key-handover.md [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+
+In the superchain-ops repo, tasks live in `tasks/<NETWORK_DIR>/<RUNBOOK_DIR>` where:
+
+- `NETWORK_DIR` is `eth` for Ethereum mainnet and `sep` for Sepolia.
+- `RUNBOOK_DIR` is of the form `{chainName}-{upgradeIndex}-{upgradeName}`.
+    - `chainName` is just the chain’s name i.e. `base` . This is excluded for OP Chains.
+    - `upgradeIndex` starts at `001` for the first playbook and increments each time. This gives a sequential ordering to upgrade transactions occurring on that chain.
+    - `upgradeName` is `key-handover`
+
+#### Copy the following files into the task directory
```
Comment (blmalone) [inline runbooks/key-handover/key-handover.md:62]: I see why these files are in this directory. I think that you can just reference the corresponding mmz files and make a point of saying that they should modify for their networks.

File: runbooks/key-handover/key-handover.md [hunk 6]
```diff
@@ -0,0 +1,252 @@
...
+    1. You can replace `eth` in the URL with `sep` for testnet key hand overs
+    2. You can replace the address in the URL with the current `ProxyAdminOwner`
+2. Enter the address of the chain’s `ProxyAdmin` on Ethereum
+3. Add the [ProxyAdmin contract ABI](https://raw.githubusercontent.com/ethereum-optimism/optimism/4cbe59b6568b738be7c4f66fea5441f3e932a308/packages/contracts-bedrock/snapshots/abi/ProxyAdmin.json)
+4. Select `transferOwnership` from the `Contract Method Selector`
+5. Enter the address of the chain’s `ProxyAdmin` in the `_proxy(address)` field
+6. Enter the 2-of-2 Multisig Address of the Optimism Foundation and Security Council [0x5a0Aae59D09fccBdDb6C6CcEB07B7279367C3d2A](https://etherscan.io/address/0x5a0Aae59D09fccBdDb6C6CcEB07B7279367C3d2A) in the `_newAdmin(address)` field
+    1. This is the same as OP Mainnets: https://github.com/ethereum-optimism/superchain-registry/blob/c01722001e88a8d21a6450f89bb7bb42311c9609/superchain/configs/mainnet/op.toml#L50
+7. Click `Add transaction` and then Click `Create Batch`
+   1. ![Add TX and Create Batch](./add-tx-create-batch.png)
```
Comment (blmalone) [inline runbooks/key-handover/key-handover.md:95]: these images are a nice touch 👍🏻

**Review by blmalone — COMMENTED** [review]
lgtm! Just some questions around whether we want to duplicate code vs reusing code that's been through the standard code review process with many eyes on it.

File: runbooks/key-handover/key-handover.md (hunk 6 above)
Comment (sbvegan) [reply runbooks/key-handover/key-handover.md:95]: thanks!

**Review by blmalone — APPROVED** [review]

**Pull request merged**